### PR TITLE
Increase query pool size

### PR DESCRIPTION
**Pull request by huwb**
Fix for #662 

@skaughtx0r will this fix the issue do you think? I've bumped pool size instead of max requests as a first step.

### CODE REVIEW

**Comment by skaughtx0r** [comment]
@huwb This fix does not work :frowning_face:

**Comment by skaughtx0r** [comment]
@huwb I just tested this on another Linux desktop computer that's running an NVIDIA GTX 970 and I have the same problem. (My previous tests were on a laptop with a Nvidia Quadro M3000M)

**Comment by huwb** [comment]
Thanks @skaughtx0r , updated

File: crest/Assets/Crest/Crest/Scripts/Collision/QueryBase.cs [hunk 1]
```diff
@@ -25,7 +25,8 @@ public abstract class QueryBase
         protected abstract string QueryShaderName { get; }
         protected abstract string QueryKernelName { get; }
 
-        const int s_maxRequests = 4;
+		// 4 was enough for a long time, but Linux setups seems to demand 7
```
Comment (daleeidd) [inline crest/Assets/Crest/Crest/Scripts/Collision/QueryBase.cs:28]: Comment indentation whitespace is tabs when file is spaces.

**Review by daleeidd — CHANGES_REQUESTED** [review]

**Comment by daleeidd** [comment]
#605 might also be fixed with this PR. Just linking the issue here.

File: crest/Assets/Crest/Crest/Scripts/Collision/QueryBase.cs (hunk 1 above)
Comment (huwb) [reply crest/Assets/Crest/Crest/Scripts/Collision/QueryBase.cs]: Thanks

**Comment by huwb** [comment]
I assume that was the only feedback so i'm going to merge @daleeidd so i can incroporate it in the next release today. Hope thats ok!

**Pull request merged**